src/providers/twitch/TwitchChannel.cpp

```diff
@@ -164,7 +136,6 @@ TwitchChannel::TwitchChannel(const QString &name)
 void TwitchChannel::initialize()
 {
     this->fetchDisplayName();
-    this->refreshChatters();
```

Comment:
Why do we no longer want to refresh chatters on startup?

Reply:
Same as the timer essentially. If we need to make sure that mods have an updated chatter list it can be reverted.

Reply:
Mods should probably always have access to an accurate @ list. Reverted code.